src/rpc/masternode.cpp

```diff
@@ -324,6 +324,138 @@ UniValue masternode_winners(const JSONRPCRequest& request)
 
     return obj;
 }
+void masternode_payments_help()
+{
+    throw std::runtime_error(
+            "masternode payments ( \"blockhash\" count )\n"
+            "\nReturns an array of deterministic masternodes and their payments for the specified block\n"
+            "\nArguments:\n"
+            "1. \"blockhash\"                       (string, optional, default=tip) The hash of the starting block\n"
+            "2. count                             (numeric, optional, default=1) The number of blocks to return.\n"
+            "                                     Will return <count> previous blocks if <count> is negative.\n"
+            "                                     Both 1 and -1 correspond to the chain tip.\n"
+            "\nResult:\n"
+            "  [                                  (array) Blocks\n"
+            "    {\n"
+            "       \"height\" : n,                 (numeric) The height of the block\n"
+            "       \"blockhash\" : \"hash\",         (string) The hash of the block\n"
```

Comment:
should we include a global amount that is the sum of each payee's amount?

Reply:
Per block or per masternode? or both?

Reply:
Both wouldn't be bad in my mind

Reply:
Well, I guess now that I think about it, don't those values have to be the same, since there is only ever one masternode per block??

Reply:
Technically, nothing is stoping us from paying multiple MNs in one block, it's just not implemented (yet?) :)

Reply:
Okie dokie, I'm fine with it then :D